src/charm.py

```diff
@@ -134,19 +137,28 @@ def _on_database_created(self, event: DatabaseCreatedEvent) -> None:
             self.unit.status = WaitingStatus("Waiting for storage to be attached")
             event.defer()
             return
+        self._write_database_information_in_config_file(event.uris)
```

Comment:
EventBase doesn't have the URI, does it?

Reply:
done